python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--inputfilePath', type=str, default='', help='ranklib input file path')
```

Comment:
Consider renaming this `input-file`, `inputFile`, or `inputFilePath` or input-filepath etc. I mean camel-case or no came-case, a consistent pattern! If you use `input-file` (my preferred solution as it is less verbose) `argparse` will transform it to `parser.input_file` for free. You don't need the default! Same below!

Reply:
I prefer [snake case](https://en.wikipedia.org/wiki/Snake_case#:~:text=Snake%20case%20(stylized%20as%20snake_case,%2C%20function%2C%20and%20file%20names.) for all python code.

Reply:
yes! I prefer it too for inline code, for args i prefer "-" which results in snake-case with argparser. Anyway! Consistency is the key!